src/parser/parser.js

```diff
@@ -47,21 +47,37 @@ const grammar = {
     statement: [
       ['shortcut', '$$ = $1;'],
       ['command', '$$ = $1;'],
+      ['jump', '$$ = $1;'],
       ['link', '$$ = $1;'],
       ['assignment', '$$ = $1;'],
       ['Text', '$$ = new yy.TextNode($1, @$);'],
     ],
 
-    link: [
-      ['OptionStart Text OptionEnd', '$$ = new yy.LinkNode($2, undefined, @$);'],
-      ['OptionStart Text OptionDelimit Identifier OptionEnd', '$$ = new yy.LinkNode($2, $4, @$);'],
-    ],
-
     shortcut: [
       ['ShortcutOption Text Indent statements Dedent', '$$ = new yy.DialogOptionNode($2, $4, @$);'],
       ['ShortcutOption Text BeginCommand If expression EndCommand Indent statements Dedent', '$$ = new yy.ConditionalDialogOptionNode($2, $8, $5, @$);'],
     ],
 
+    command: [
```

Comment:
why are these added here? I would assume you only need a new entry for the jump

Reply:
The link array should not have the first element, which is the pattern used by the jump. The other moves were done for readability. Previously, the other statements were scattered around and so I moved them to one spot and ordered them as they are listed in the statement array.